src/main/java/com/blackduck/integration/detect/configuration/enumeration/ExitCodeType.java

```diff
@@ -16,7 +16,7 @@ public enum ExitCodeType {
     ),
     FAILURE_BLACKDUCK_FEATURE_ERROR(
         11,
-        "Detect encountered an error while attempting an operation on Black Duck. Ensure that your Black Duck version is compatible with this version of Detect, and that your Black Duck user account has the required roles."
+        "Detect encountered an error while attempting an operation on Black Duck. Ensure that your Black Duck version is compatible with this version of Detect, and that your Black Duck user account has the required roles, and the project version exists in Black Duck when using BOM_COMPARE mode in a rapid scan."
```

Comment:
Suggest tweak:
"While trying to perform an operation on Black Duck SCA, Detect encountered an error. Ensure that your Black Duck SCA version is compatible with this version of Detect, your Black Duck user account has the required roles, and the project version exists in Black Duck when using BOM_COMPARE mode in a rapid scan."